sopel_modules/youtube/youtube.py

```diff
@@ -27,6 +30,8 @@
 API = None
 
 
+num_retries = 5
```

Comment:
Two blank lines after, one before, please. (Really, it's two blank lines before the class definition, but this is the line you changed. :grin: